torch/csrc/jit/tensorexpr/cuda_codegen.cpp

```diff
@@ -963,11 +957,6 @@ void CudaCodeGen::CompileToNVRTC(
   int major, minor;
   getMajorMinor(prop, major, minor);
 
-#if DEBUG_PRINT
```

Comment:
Do we not need this in debug logs?

Reply:
Seems kind of low-value to me, unless someone really wants it I was inclined to remove it for the sake of spamming the console less.